packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py

```diff
@@ -74,6 +74,11 @@ class SpanAttributes:
     LLM_OPENAI_API_VERSION = "gen_ai.openai.api_version"
     LLM_OPENAI_API_TYPE = "gen_ai.openai.api_type"
 
+    # Anthropic
+    LLM_ANTHROPIC_CACHE_CREATION_INPUT_TOKENS = "llm.anthropic.usage.cache_creation_input_tokens"
+    LLM_ANTHROPIC_CACHE_READ_INPUT_TOKENS = "llm.anthropic.usage.cache_read_input_tokens"
+    LLM_ANTHROPIC_TOTAL_INPUT_TOKENS = "llm.anthropic.usage.total_input_tokens"
```

Comment:
The logic I followed when naming this attributes was the following:
1. https://github.com/traceloop/openllmetry/pull/1858#discussion_r1720759231 this comment
2. `llm` is apparently less standardized than `gen_ai`, so we are free to experiment here
3. I haven't found a separate documentation for `gen_ai.anthropic` attributes like they have for `gen_ai.openai` [here](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/gen-ai/openai.md)

So I thought, a good in between would be `llm.anthropic.usage.*`, but I am open to additional thoughts.

Also, I am not sure if restricting this to anthropic is a good idea, because both google-generativeai and openai have at least some overlap with this.

Reply:
I'd argue these should be `gen_ai` and without `anthropic`. Most of the "official" semantic conventions came after OpenLLMetry and were inspired but the work we've done here - so we should set the preferred path moving forward!

Reply:
I agree generally, but given the context I provide in https://github.com/traceloop/openllmetry/pull/2175#discussion_r1809080693 and the fact that what used to be called `gen_ai.usage.prompt_tokens` is now apparently called `gen_ai.usage.input_tokens` ["officially"](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/gen-ai/gen-ai-spans.md), we need to decide what to call these, especially the `TOTAL_INPUT_TOKENS` one.

So, for context, OpenAI does not return `cache_creation_input_tokens` as they don't charge for the operation. Their usage block looks like this:
```
usage: {
  total_tokens: 2306,
  prompt_tokens: 2006,
  completion_tokens: 300,
  
  prompt_tokens_details: {
    cached_tokens: 1920,
    audio_tokens: 0,
  },
  completion_tokens_details: {
    reasoning_tokens: 0,
    audio_tokens: 0,
  }
}
```

Gemini returns the number of `cache_creation_input_tokens` from the `cache.create` call, and they charge for storage of cache tokens per hour.

So I think, overall the formula is this:
```
input_tokens = cached_tokens + uncached_tokens
cached_tokens = cache_read_tokens # generally
cached_tokens = cache_read_tokens OR cache_creation_tokens # for Anthropic
```
and `cache_read_tokens` and `cache_creation_tokens` are charged for at vastly different prices.

With this in mind, the main open question is what are `gen_ai.usage.prompt_tokens` and `gen_ai.usage.input_tokens` bound to mean? Is it total input tokens or is it the number of uncached tokens charged for regular price? While the former makes more sense to me, I think this will sort of break any cost calculation implementations depending on this library?

Reply:
Done! @nirga please check my latest commit